README.md

```diff
@@ -224,6 +224,104 @@ messages with an unrecognised conversation ID on a dead letter queue.
 
 An example daisy chaining environment is provided in test-suite/daisy-chaining 
 
+## Endpoints
+
+The Patient Switching adaptors facade exposes two endpoints.
+
+- POST /Patient/$gpc.migratestructuredrecord
+- POST /$gpc.ack
+
+### /Patient/$gpc.migratestructuredrecord
+
+The migratestructuredrecord endpoint is the primary endpoint for the adaptor and is used to start an electronic health record transfer.
+The following is required to call this endpoint...
+
+- TO_ASID : The ASID identifier of the winning New Market Entrant (NMW)
+- FROM_ASID : The ASID identifier of the losing incumbent
+- TO_ODS : The ODS identifier of the winning NME
+- FROM_ODS : the ODS identifier of the losing incumbent
+- ConversationId : A unique GUID for each request; if you do not provide one, the adaptor will create one and return it in the response headers. It must be used for all further calls for the patient's NHS number.
+
+The endpoint also requires a JSON body that includes the needed patient NHS number. The format of the body should look like the following...
+
```

Comment:
I think code should be in a code block (three backticks) otherwise markdown will remove the whitespace